frontends/.yarnrc.yml

```diff
@@ -0,0 +1,19 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.1.1.cjs
+
+# https://github.com/vitejs/vite-plugin-react-swc/issues/74#issuecomment-1520484130
+# https://github.com/swc-project/swc/issues/5616#issuecomment-1265639797
+# This setting ensures we always install the Linux binaries when running `yarn install`. This is needed for running
+# swc natively in Docker (addresses swc-loader bindings not found error).
+supportedArchitectures:
+  os:
+    - current
+    - linux
```

Comment:
This is nice. After yarn installing on my host (as well as on docker up) I was able to compile files in both docker and on host. Running the tests is a lot faster on host.